docs/paas-plus/dev/check-spryker-code-upgrader-logs.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Check Spryker Code Upgrader logs
+description: Instructions for checking Spryker Code Upgrader logs
+template: howto-guide-template
+---
+
+To check the logs of Spryker Code Upgrader, do the following:
+
+1. In Spryker CI, go to **Projects**.
+2. On the **Projects**, select **Spryker Code Upgrader**.
+3. On the **Pipelines** page, select the **Upgrader** pipeline.
+4. In the **Runs** tab, select the run you want to check the logs for.
```

Comment:
```suggestion
4. In the **Runs** tab, select the run you want to check the logs for.
```

Reply:
What's the difference?

Reply:
@AlexSlawinski

Reply:
Ah no difference here, resolving it @DmytroKlymanSpryker. Left-over I forgot to remove :)